portability/CMakeLists.txt

```diff
@@ -18,14 +18,13 @@ add_library(${LIBTOKUPORTABILITY} SHARED ${tokuportability_srcs})
 target_link_libraries(${LIBTOKUPORTABILITY} LINK_PRIVATE ${LIBJEMALLOC})
 target_link_libraries(${LIBTOKUPORTABILITY} LINK_PUBLIC ${CMAKE_THREAD_LIBS_INIT} ${EXTRA_SYSTEM_LIBS})
 
-add_library(tokuportability_static_conv STATIC ${tokuportability_srcs})
-set_target_properties(tokuportability_static_conv PROPERTIES POSITION_INDEPENDENT_CODE ON)
-add_dependencies(tokuportability_static_conv build_jemalloc)
-set(tokuportability_source_libs tokuportability_static_conv ${LIBJEMALLOC} ${CMAKE_THREAD_LIBS_INIT} ${EXTRA_SYSTEM_LIBS})
-toku_merge_static_libs(${LIBTOKUPORTABILITY}_static ${LIBTOKUPORTABILITY}_static "${tokuportability_source_libs}")
```

Comment:
It seems that the sole uses of toku_merge_static_libs have been removed, thus please remove the macro itself too